packages/react-scripts/template/src/index.css

```diff
@@ -1,5 +1,12 @@
+* {
+  box-sizing: border-box;
```

Comment:
I would consider it essentially, a relative to normalize. Better [document](http://v4-alpha.getbootstrap.com/content/reboot/) perhaps though